book/graphs/other/brg01.qmd

```diff
@@ -273,16 +283,19 @@ anl <- anl[!duplicated(anl$USUBJID), ] %>%
   mutate(N = as.numeric(N)) %>%
   mutate(pct = round((n / N) * 100, 2))
 
-ggplot(anl, aes(x = .data[["ACTARM"]], fill = .data[["AETOXGRC"]])) +
+plot <- ggplot(anl, aes(x = .data[["ACTARM"]], fill = .data[["AETOXGRC"]])) +
   geom_col(aes(y = pct)) +
   guides(fill = guide_legend(title = getElement(vl_ae, "AETOXGRC"))) +
   scale_y_continuous(labels = scales::comma) +
   theme(plot.title = element_text(hjust = 0.5)) +
   ggtitle("Percent of Subjects With AE by Highest \n NCI CTCAE Grade: Safety-Evaluable Subjects") +
   xlab(paste0(getElement(vl_ae, "ACTARM"), " (N = ", denom, ")")) +
   ylab("Percent")
+plot
 ```
 
+{{< include ../../test-utils/save_results.qmd >}}
+
 ## Teal
 
 ```{r teal, message=FALSE}
```

Comment:
if speed of tests is a concern you might also add exclude teal chunk from test run

Reply:
The `teal` part of the quarto document renders instantaneously so this does not affect the overall runtime. What was helpful was to delete the unwanted docs and just render and test the required docs.